apps/fluent-tester/src/E2E/CheckboxExperimental/specs/ExperimentalCheckbox.spec.win.ts

```diff
@@ -10,7 +10,7 @@ import { PAGE_TIMEOUT, BOOT_APP_TIMEOUT, CHECKBOX_A11Y_ROLE } from '../../common
 describe('Experimental Checkbox Testing Initialization', () => {
   it('Wait for app load', () => {
     NavigateAppPage.waitForPageDisplayed(BOOT_APP_TIMEOUT);
-    expect(NavigateAppPage.isPageLoaded()).toBeTruthy();
+    expect(NavigateAppPage.isPageLoaded()).toBeTruthy(ExperimentalCheckboxPageObject.ERRORMESSAGE_APPLOAD);
```

Comment:
>ExperimentalCheckboxPageObject [](http://example.com/codeflow?start=54&length=30)

SHould this be NavigateAppPage for consistency with other pages?

I'm guessing in general it doesn't matter where it comes from, that ExperimentalCheckboxPageObject is inheriting ERRORMESSAGE_APPLOAD from some BaseAppPage, but it seems like you're trying to not depend on the individual test pages for the first load test?

Reply:
Good catch! Changing it now